kotlintest-runner/kotlintest-runner-jvm/src/main/kotlin/io/kotlintest/runner/jvm/internal/topLevelTests.kt

```diff
@@ -0,0 +1,33 @@
+package io.kotlintest.runner.jvm.internal
```

Comment:
I feel that this function could be placed near `Spec`, as it's completely related to `Spec`.
I would also add it as a member function of the `Spec` class ,with `Spec` as receiver, but I don't think that's needed now.